collector/slave_status.go

```diff
@@ -61,9 +76,17 @@ func ScrapeSlaveStatus(db *sql.DB, ch chan<- prometheus.Metric) error {
 		for i, col := range slaveCols {
 			if value, ok := parseStatus(*scanArgs[i].(*sql.RawBytes)); ok { // Silently skip unparsable values.
 				ch <- prometheus.MustNewConstMetric(
-					newDesc(slaveStatus, strings.ToLower(col), "Generic metric from SHOW SLAVE STATUS."),
+					prometheus.NewDesc(
```

Comment:
Hmmm. What is the purpose of `newDesc()` then? It doesn't seem to do much, and is not that widely used apart from engine_toukdb, global_status and global_variables.